apps/studio/components/interfaces/Auth/Policies/PolicyEditorModal/PolicyEditorModal.constants.ts

```diff
@@ -173,3 +173,143 @@ for select using (
+on realtime.messages for select
+using ( realtime.messages.extension = 'broadcast' AND realtime.topic() = 'channel_name' );`.trim(),
+      name: 'Enable listening to broadcasts from a specific channel',
+      definition: `realtime.messages.extension = 'broadcast' AND realtime.topic() = 'channel_name'`,
+      check: '',
+      command: 'SELECT',
+      roles: [],
+    },
+    {
+      id: 'policy-broadcast-4',
+      preview: false,
+      templateName: 'Enable pushing broadcasts to specific channel',
+      description: 'This policy enables pushing broadcasts to specific channel.',
+      statement: `
+create policy "Enable pushing broadcasts to specific channel"
+ON realtime.messages for update
+using ( realtime.messages.extension = 'broadcast' AND realtime.topic() = 'channel_name' )
+with check ( realtime.messages.extension = 'broadcast' AND realtime.topic() = 'channel_name' );`.trim(),
+      name: 'Enable pushing broadcasts to specific channel',
+      definition: `realtime.messages.extension = 'broadcast' AND realtime.topic() = 'channel_name'`,
+      check: `realtime.messages.extension = 'broadcast' AND realtime.topic() = 'channel_name''`,
+      command: 'UPDATE',
+      roles: [],
+    },
+    {
+      id: 'policy-presences-1',
+      preview: false,
+      templateName: 'Enable listening to presences on all channels for authenticated users only',
+      description:
+        'This policy enables listening to presences on all channels for all authenticated users only.',
+      statement: `
+create policy "Enable listening to presences on all channels for authenticated users only"
+on realtime.messages for select
+to authenticated
+using ( realtime.messages.extension = 'presence' );`.trim(),
+      name: 'Enable listening to presences on all channels for authenticated users only',
+      definition: "realtime.messages.extension = 'presence'",
+      check: '',
+      command: 'SELECT',
+      roles: ['authenticated'],
+    },
+    {
+      id: 'policy-presences-2',
+      preview: false,
+      templateName: 'Enable broadcasting presences on all channels for authenticated users only',
+      description:
+        'This policy enables broadcasting presences on all channels for all authenticated users only.',
+      statement: `
+create policy "Enable broadcasting presences on all channels for authenticated users only"
+ON realtime.messages for update
+TO authenticated
+using ( realtime.messages.extension = 'presence' )
+with check ( realtime.messages.extension = 'presence' );
+  ;`.trim(),
+      name: 'Enable broadcasting presences on all channels for authenticated users only',
+      definition: "realtime.messages.extension = 'presence'",
+      check: "realtime.messages.extension = 'presence'",
+      command: 'UPDATE',
+      roles: ['authenticated'],
+    },
+    {
+      id: 'policy-presences-3',
+      preview: false,
+      templateName: 'Enable listening to presences from a specific channel',
```

Comment:
"Enable" doesn't sound like the right verb here - wondering if something like "Listen to presences from a specific channel only" might be better?

Reply:
yeah... "Listen to presence ..."  or "Allow listening to presence..." ?

Reply:
looking at this in the UI, maybe 

"Allow listening for presence..." 

Changes:  
enable → allow
to → for 

![screenshot-2024-06-25-at-12 46 57](https://github.com/supabase/supabase/assets/105593/fbd7eb4e-637c-46fd-a7a6-3dc8f2f6d8c1)

Reply:
Fixed.